curriculum/challenges/english/08-coding-interview-prep/rosetta-code/self-describing-numbers.english.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Self Describing Numbers
+id: 5eaf48389ee512d4d103684b
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+There are several so-called "self describing" or <a href="https://en.wikipedia.org/wiki/Self-descriptive_number">"self-descriptive"</a> integers. <br />
```

Comment:
Let's remove the line break and add `target="_blank"` here to open the link in a new tab:

```
There are several so-called "self describing" or <a href="https://en.wikipedia.org/wiki/Self-descriptive_number" target="_blank">"self-descriptive"</a> integers.
```